helpers/inlineExec.ts

```diff
@@ -0,0 +1,18 @@
+import { exec } from "@actions/exec";
+
+/**
+ * By default, await exec() returns the response number of the command.
+ * This returns the stdout data instead.
+ * @param commandLine The command to run
+ */
+export const inlineExec = async (commandLine: string) => {
+  let stdoutData = "";
+  await exec(commandLine, undefined, {
+    listeners: {
+      stdout: (data: Buffer) => {
+        stdoutData += data.toString();
+      },
```

Comment:
Those are all good points! Il will have to test that out indeed.

Reply:
I've added `failOnStdErr: true`. Since from the doc it will `fail if output to stderr`, I don't think that we need to add `return [stdoutData, stdErrData]`.
Plus, by returning only the stdout, it makes it easier to `inline` it:
```js
JSON.parse(await inlineExec(`aws ...`))
```